elixir/apps/domain/lib/domain/auth/adapters/google_workspace/api_client.ex

```diff
@@ -230,6 +230,10 @@ defmodule Domain.Auth.Adapters.GoogleWorkspace.APIClient do
 
         {:error, :invalid_response}
 
+      # This is expected if the group has no members or we're on the last page
+      :error when key == "members" ->
+        default_if_missing
```

Comment:
If we're pulling out the expected missing value here, should we make the other `:error` case not return the default any more?

Reply:
👍